learn-regular-expressions-by-building-a-spam-filter/spamFilter060.js

```diff
@@ -0,0 +1,27 @@
+const isSpam = msg => {
+  const blacklistRegexes = [
```

Comment:
This lesson could also be broken up. You could start by explaining that a single line of regex is not easy to read/maintain, and that an array to store everything would be better. Then users could build up the `blacklistRegexes` over several lessons before you explain a little about the `some` array. Your example code would be a good way to show `some` in action rather than write about the callback argument and truthy values.